quests/variables_and_templates.md

```diff
@@ -17,15 +17,15 @@ configuration file and service unit file are both managed by static source
 files in the module's `files` directory.
 
 In this quest, we introduce **variables** and **templates**. Variables and
-templates, let separate data from the resources and files you use to define
+templates allow separate data from the resources and files you use define
```

Comment:
This whole section needs a little more work to make sense:

In this quest, we introduce **variables** and **templates**. Once you assign a value to a variable in a Puppet manifest, you can use that variable throughout the manifest to yield the assigned value. Through templates, you can incorporate these variables into the content of any files your Puppet manifest manages.

While it's convenient to introduce variables and templates here as a pair, as you work through the rest of this quest guide, you'll see that variables are used along with many of Puppet's other features to help write adaptable Puppet code.